### PR TITLE
Added files of updated CV

### PR DESCRIPTION
**Pull request by olalipa**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Encode+Sans+Condensed&display=swap" rel="stylesheet">
+  <meta charset="UTF-8">
+  <title>CV</title>
+</head>
+<body>
+<div class="wrapper">
+  <header class="header">
+    <div class="photo">
+      <img src="photo1.png">
```
Comment (Solnick) [inline index.html]: remember to add `alt` attribute to the img

File: photo1.png
Comment (Solnick) [inline photo1.png:1]: 🤣

File: index.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Encode+Sans+Condensed&display=swap" rel="stylesheet">
+  <meta charset="UTF-8">
+  <title>CV</title>
+</head>
+<body>
```
Comment (Solnick) [inline index.html]: I think you can use more semantic tags in this file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        </div>
+        <div class="skills-container">
+          <div class="elit">Consectetur elit</div>
+          <div class="progress-bar-back">
+            <div class="progress-bar-front90"></div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="qualifications">
```
Comment (Solnick) [inline index.html]: Remember about proper indentation

File: index.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+      <div class="experience">EXPERIENCE</div>
+      <div class="job-position">Sales Force Team Leader (2006 - NOW)</div>
+      <div class="experience-text">Sed et tincidunt velit, a condimentum orci. Sed quis pellentesque nisl. Maecenas et ullamcorper nulla, ut vehicula augue. Nam at ex lacinia, accumsan nunc in, ultricies nunc.
+      </div>
+      <div class="job-position">Sales Manager (2003 - 2006)</div>
+      <div class="experience-text">Mauris porttitor mauris a nunc elementum, ut consequat libero tincidunt. Curabitur egestas sit amet tellus nec blandit.
+      </div>
+      <div class="job-position">Sales Person (1999 - 2003)</div>
+      <div class="experience-text"> Duis iaculis sem id massa bibendum, et auctor nibh tincidunt. Nulla velit metus, porta ac nulla ac, rhoncus pulvinar eros.
```
Comment (Solnick) [inline index.html]: this space is not needed
```suggestion
      <div class="experience-text">Duis iaculis sem id massa bibendum, et auctor nibh tincidunt. Nulla velit metus, porta ac nulla ac, rhoncus pulvinar eros.

```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+          </div>
+        </div>
+        <div class="skills-container">
+          <div class="developer">Web Develop</div>
+          <div class="progress-bar-back">
+            <div class="progress-bar-front90"></div>
+          </div>
+        </div>
+        <div class="skills-container">
+          <div class="lorem">Lorem Ipsum</div>
```
Comment (Solnick) [inline index.html]: You added a class for every possible element in the HTML, while you are not using all of them. Please get rid of the redundant classes.

**Review by Solnick — CHANGES_REQUESTED** [review]
bez nazwy (1)

**Review by Solnick — CHANGES_REQUESTED** [review]
You didn't fixed the HTML file according to comments. Also there is a visual flaw (screenshot), the picture and the container under are not aligned 
image

File: index.html (hunk 1 above)
Comment (olalipa) [reply index.html]: Added

File: index.html (hunk 4 above)
Comment (olalipa) [reply index.html]: Done

File: index.html (hunk 3 above)
Comment (olalipa) [reply index.html]: Corrected

File: index.html (hunk 5 above)
Comment (olalipa) [reply index.html]: Reduced:)

**Review by Solnick — CHANGES_REQUESTED** [review]
image
Please set the maximum width for the whole CV. It should not be this wide on bigger screens. Maybe set max width to 1200px…

**Review by Solnick — APPROVED** [review]

**Pull request merged**